draft-foudil-securitytxt.md

```diff
@@ -112,17 +117,15 @@ HTTPS. Security email addresses SHOULD use the conventions defined
 in section 4 of {{!RFC2142}}, but there is no requirement for this directive
 to be an email address.
 
-While URIs already include the ability to have both email address and phone
-numbers via "mailto" and "tel" prefixes, allowing this information to be listed
-without a prefix is intended for ease of use and readability.
+The value MUST follow the general syntax described in {{!RFC3986}}. This means that "mailto" and "tel" URI schemes are required.
```

Comment:
I wonder if the wording should be "This means that "mailto" and "tel" URI schemes **MUST be used when specifying email addresses and telephone numbers"**